service/swagger.js

```diff
@@ -39,7 +39,8 @@ const Team = Joi.object({
   path: Joi.string(),
   users: Joi.array().items(UserRef),
   policies: Joi.array().items(PolicyRef),
-  organizationId: Joi.string()
+  organizationId: Joi.string(),
+  membersCount: Joi.number()
```

Comment:
is this same schema used for the single get? are we returning the membersCount on that too or should be create two separate team definitions, one for the single get and one for the listing?